Recipe/Recipe/Utils/ValidationService.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ValidationService.swift
+//  Recipe
+//
+//  Created by user on 09/08/2024.
+//
+
+import Foundation
+
+class ValidationService {
+    static func validateEmail(_ email: String) -> Bool {
```

Comment:
please create protocol for email validation and password validation

protocol ValidationService { }
class EmailValidationService: ValidationService { }
class PasswordValidationService: ValidationService { }